crates/viona-api/src/ffi.rs

```diff
@@ -0,0 +1,145 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![allow(non_camel_case_types)]
+
+use libc::size_t;
+use std::ffi::c_void;
+
+const fn vna_ioc(ioc: i32) -> i32 {
+    const V: i32 = b'V' as i32;
+    const C: i32 = b'C' as i32;
+    V << 16 | C << 8 | ioc
+}
```

Comment:
my  assumption upon reading this code (and perhaps I am wrong here) is that `ioc` will always be 8 bits wide, since everything above that will be clobbered, is that true? and if so, perhaps we ought to take it as a `u8` here?